pkg/http/api.go

```diff
@@ -1879,11 +1885,20 @@ func (a *api) onShutdown(reqCtx *fasthttp.RequestCtx) {
 }
 
 func (a *api) onPublish(reqCtx *fasthttp.RequestCtx) {
+	a.onPubSubPublish(reqCtx, "", "")
+}
+
+func (a *api) onActorPublishAlpha1(reqCtx *fasthttp.RequestCtx) {
```

Comment:
nit: just onActorPublish is enough since internal method name can change freely.